File: Typescript/test/findPoisonedDuration.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/findPoisonedDuration';

test('Example 1', () => {
	let input0 = [1, 4];
	let input1 = 2;
	let expected = 4;
	expect(solution.findPoisonedDuration(input0, input1)).toEqual(expected);
});

test('Example 2', () => {
	let input0 = [1, 2];
	let input1 = 2;
	let expected = 3;
	expect(solution.findPoisonedDuration(input0, input1)).toEqual(expected);
});

File: Dart/test/findPoisonedDuration_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/findPoisonedDuration.dart';

void main() {
  test("test example 1", () {
    var input0 = [1, 4];
    var input1 = 2;
    var expected = 4;

    final sln = Solution();
    var output = sln.findPoisonedDuration(input0, input1);
    expect(output, expected);
  });

  test("test example 2", () {
    var input0 = [1, 2];
    var input1 = 2;
    var expected = 3;

    final sln = Solution();
    var output = sln.findPoisonedDuration(input0, input1);
    expect(output, expected);
  });
}
